tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -560,3 +560,66 @@
 - name: Adopted Neutron and OVN agents post-checks
   ansible.builtin.include_tasks:
     file: neutron_verify.yaml
+
+- name: Remove leftover OOO services
```

Comment:
We can land this as-is i think, to start testing with the new role, but subsequently i think we should make two changes:

* Try putting this before adopting the data plane, rather than after (we don't want to be running new & old services at the same time).
* Add this into the documentation as well.